pkg/frontend/tgt/frontend.go

```diff
@@ -19,14 +19,17 @@ const (
 type Tgt struct {
 	s *socket.Socket
 
-	isUp         bool
-	dev          longhorndev.DeviceService
-	frontendName string
+	isUp                      bool
+	dev                       longhorndev.DeviceService
+	frontendName              string
+	scsiTimeout               int64
+	iscsiAbortTimeout         int64
+	iscsiTargetRequestTimeout int64
```

Comment:
Use `time.Duration`, then use the right method to convert to second.

Reply:
Addressed. Thank you.